http/jsonapi/runtime/parameters.go

```diff
@@ -90,6 +91,33 @@ func ScanParameters(w http.ResponseWriter, r *http.Request, parameters ...*ScanP
 			panic(fmt.Errorf("impossible scanning location: %d", param.Location))
 		}
 
+		// handle string and dates types
+		switch v := param.Data.(type) {
+		case *time.Time:
+			// zero value case - empty case
+			if len(scanData) == 0 {
+				continue
+			}
+
+			// parse value
+			ti, err := time.Parse(time.RFC3339Nano, scanData)
+			if err != nil {
+				WriteError(w, http.StatusBadRequest, &Error{
+					Title:  fmt.Sprintf("invalid time value for %s", param.Name),
+					Detail: fmt.Sprintf("time value not in RFC3339 format got: %q", scanData),
```

Comment:
RFC3339Nano is different from RFC3339 and thus should also be differentiated in the details.